package.json

```diff
@@ -15,5 +15,8 @@
   "devDependencies": {
     "mocha": "x.x.x",
     "mocha-jshint": "0.0.9"
+  },
+  "scripts": {
+    "test": "rm -f restbase && mocha"
```

Comment:
restbase is the name of database and on running each test a 'restbase' file is created. deleting the restbase file deletes all the tables.